### ISSUE TITLE
A few things

### ISSUE THREAD

**Issue by doradoro** [jliljebl/flowblade-forum#21]
- I'm unable to move Compositors from Track
- Animated GIF arent showing, shows a white image
- When i do a Luma Key and the image is not full screen, the non existent part becomes black, not transparent
- I found a very easy way to create new tracks but know i getting crazy searching how to do it
- I still did not found an easy way to trasnform-move-scale a video...

I think this is the best video editing tool in linux (never been able to use davinci) i moved from the always crashing kdenlive to the horrible transparency's Olive and then to Flowblade. Doesnt crash so much and when it does i can recover, transparencies are ok most of the time BUT have a very difficult way to work compared to Olive (cut, move, transform, etc)

**Comment by jliljebl** [comment]
Ok, there are multiple things here and giving an answer on all of them could take some time.

I'll answer a few now in separate comments.

**Comment by jliljebl** [comment]
> Having multiple "transform" choices in filters and Compositions is confusing and in the end, it would be easier to have all those choices in one single transform Composition, the same for Alphas, Chromas...

Over all we are looking to simplify workflow on this front.

I'm not sure if we can get to a design that is as approachable to new user as possible because of technical and design limitations in the application but the direction is towards a more simple UX.

**Comment by jliljebl** [comment]
>  I'm unable to move Compositors from Track

This came to be because originally MLT would crash if Compositors were moved to different tracks, but come to think of it we should have internal code to make this possible so maybe I'll give this a go.

**Comment by jliljebl** [comment]
> Animated GIFs arent showing, shows a white image

Do these work in Kdenlive (or Shotcut)? I haven't looked at this, but on first look I would think that MLT does not support these and we can have these very quickly.

Flowblade supports image sequences, so you can do GIF -> image sequence: https://askubuntu.com/questions/101526/how-can-i-split-an-animated-gif-file-into-its-component-frames/101527

**Comment by jliljebl** [comment]
> When i do a Luma Key and the image is not full screen, the non existent part becomes black, not transparent

Yes, this is something that needs to be fixed in MLT and I will look into this eventually, I have some ideas on how to improve situation here but it is impossible to say when we get improvement here. Additional contributors is the only thing that fixes this stuff, but unfortunately they do not seem to be forthcoming.

**Comment by jliljebl** [comment]
> I found a very easy way to create new tracks but now i'm getting crazy searching how i done it

*Edit -> Change Sequence Tracks Count...*

**Comment by jliljebl** [comment]
> I can't remade-refresh Compound clips

Compound clips are MLT XML media objects without any meta saved at creation time so creating "live linked" compound clips probably isn't on the cards anytime soon.

Just keep recreating them as need is current offered approach here.

**Comment by doradoro** [comment]
I just found the way to easily cut parts of videos and its much more stable than Olive so Flowblade became my main video editor for everything. It would be great to add it to the second mouse button menu.

I just did learn to copy compositors so it bacame easier.

Gifs works in all other editors but in some of them the animation doesnt works. I switched from manjaro xfce to the mate desktop and now gifs appears properly in the Clip Monitor but it gets stuck trying to use them.

It would be great to be able to use a sequences inside another sequences like if it were a normal video instead using compound clips.
